ocaml/xapi/xapi_host.mli

```diff
@@ -514,12 +514,12 @@ val set_sched_gran :
 val get_sched_gran :
   __context:Context.t -> self:API.ref_host -> API.host_sched_gran
 
-val emergency_disable_tls_verification : __context:'a -> unit
+val emergency_disable_tls_verification : __context:Context.t -> unit
```

Comment:
I'm not so sure about this

``__context:`a`` was communicating that you can't use the `__context` parameter in this function - `emergency_disable_tls_verification` and `emergency_reenable_tls_verification` are emergency comands, so we can expect that the host they are being called on cannot reach the master. In particular we can expect 'Db.X.y ~__context` to hang or fail